modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonClientExperimentalCodegen.java

```diff
@@ -910,7 +910,14 @@ public String getTypeString(Schema p, String prefix, String suffix) {
         } else if (ModelUtils.isArraySchema(p)) {
             ArraySchema ap = (ArraySchema) p;
             Schema inner = ap.getItems();
-            return prefix + "[" + getTypeString(inner, "", "") + "]" + fullSuffix;
+            if (inner == null) {
+                // Per JSON schema specification, the array "items" attribute is optional.
+                // When "items" is not specified, the elements of the array
+                // may be anything at all.
+                return prefix + "[bool, date, datetime, dict, float, int, list, str, none_type]" + fullSuffix;
```

Comment:
@spacether , can you help review this?

Reply:
So this will only happen if we are in an array, right?
Don't we instead need to cover the ModelUtils.isAnyTypeSchema(schema) case?
How about higher up at line 907 use:
```
Schema anyType = new Schema();
if (ModelUtils.isAnyTypeSchema(schema)):
  return prefix + "bool, date, datetime, dict, float, int, list, str, none_type" + fullSufix;
} else if ((ModelUtils.isMapSchema(p) || "object".equals(p.getType())) && 
  ...
} else if (ModelUtils.isArraySchema(p)) 
  ArraySchema ap = (ArraySchema) p;
  Schema inner = ap.getItems();
  if (inner == null) {
    inner = anyType;
  }
  return prefix + "[" + getTypeString(inner, "", "") + "]" + fullSuffix;
...

```

Reply:
We also need the `if (ModelUtils.isAnyTypeSchema(schema)):` in case we travel through an object(dict) to get there. In that case, our return would look like:
`{str: (bool, date, datetime, dict, float, int, list, str, none_type)}`